assets/schema_input.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://raw.githubusercontent.com/nf-core/callingcards/master/assets/schema_input.json",
+    "title": "nf-core/callingcards pipeline - params.input schema",
+    "description": "Schema for the file provided with params.input",
+    "type": "array",
+    "items": {
+        "type": "object",
+        "properties": {
+            "sample": {
+                "type": "string",
+                "pattern": "^\\S+$",
+                "errorMessage": "Sample name must be provided and cannot contain spaces"
+            },
+            "fastq_1": {
+                "type": "string",
+                "format": "file-path",
+                "exists": true,
+                "pattern": "^\\S+\\.f(ast)?q\\.gz$",
+                "errorMessage": "FastQ file for reads 1 must be provided. The filepath cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz'"
+            },
+            "fastq_2": {
+                "type": "string",
+                "format": "file-path",
+                "errorMessage": "FastQ file for reads 2 cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz'",
```

Comment:
You could have a 'dependentOn' on clause here, where if you supply `fastq2`, you must also have a `fastq`

Reply:
e.g.: https://github.com/nf-core/eager/blob/62f1b579a0b3de82f7590f85e5c4407a7de775a8/assets/schema_input.json#L67

Reply:
Although I know it's redundant to the `required` below

Reply:
Added -- thank you!